src/stac_utils/secret_context.py

```diff
@@ -11,6 +11,14 @@ def secrets(
     aws_region: str = None,
     dictionary: dict = None,
 ):
+    """
```

Comment:
would describe this as "Takes either a file, a Python dict, or an AWS secret in Secrets Manager and loads it all into the os.environ as a context. Usage is typically:

```
with secrets('secrets.json'):
  rest_of_code_here
```
"